src/public/js/main.js

```diff
@@ -6,6 +6,8 @@
 
   function main(CONSTANTS) {
     var socket = io();
+    var n = 255;
+    var k = 245;
```

Comment:
Each client's main should have an n and a k, eventually -- for now, encode / decode should take n & k and n & k should be defined in main